acncli/deployment/manager-master.yaml

```diff
@@ -16,23 +16,24 @@ spec:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
               - matchExpressions:
-                  - key: node-role.kubernetes.io/agent
+                  - key: node-role.kubernetes.io/master
                     operator: Exists
       tolerations:
         - effect: NoSchedule
           key: node-role.kubernetes.io/master
           operator: Exists
       hostNetwork: true
-      imagePullSecrets:
-        - name: acndev.azurecr.io-key
       containers:
         - name: azure-cni-installer
-          image: acndev.azurecr.io/azure-cni-installer:07f84fcdbef4
+          image: mcr.microsoft.com/containernetworking/azure-cni-manager:v1.1.8-4-g4d14734-dirty-3
+          imagePullPolicy: Always
           env:
             - name: CNI_OS
               value: linux
             - name: CNI_TYPE
               value: singletenancy
+            - name: CNI_MODE
```

Comment:
Two different naming conventions for envs in agent and master, agent has "Azure_*".